pages/eluna/Creature/GetRank.html

```diff
@@ -0,0 +1,144 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta name="description" content="API documentation for the Creature:GetRank method in the Eluna engine.">
+    <meta name="keywords" content="eluna, lua, lua engine, azerothcore, script, scripting, doc, docs, documentation">
+
+    <title>Creature:GetRank - Eluna</title>
+
+    <link rel="stylesheet" type="text/css" href="../static/main.css">
+    <link rel="stylesheet" type="text/css" href="../static/dark.css">
+
+    <link rel="shortcut icon" href="../static/favicon.ico">
+
+</head>
+<body class="rustdoc">
+<!--[if lte IE 8]>
+    <div class="warning">
+        This old browser is unsupported and will most likely display funky
+        things.
+    </div>
+    <![endif]-->
+
+<section class="sidebar">
+    <a href='../index.html'><img src='../static/eluna-logo.png' alt='Eluna Logo' width='100'></a>
+    <div class="block">
+        
+    <h2>Creature Methods</h2>
+    <script src="sidebar.js"></script>
+    <script>
+      // Highlight current method by adding "current" class to it
+      var element = document.getElementById("Creature:GetRank");
+      if (element) {
+        var classname = "current";
+        arr = element.className.split(" ");
+        if (arr.indexOf(classname) == -1) {
+          element.className += " " + classname;
+        }
+      }
```

Comment:
not really relevant but you could write it in the following way:
```javascript
      const element = document.getElementById("Creature:GetRank");
      if (element) {
        const classname = "current";
        const arr = element.className.split(" ");
        if (arr.indexOf(classname) == -1) {
          element.className += ` ${classname}`;
        }
      }
```

Reply:
These documentation PRs are automated, the template is here: https://github.com/azerothcore/mod-eluna/blob/master/src/LuaEngine/docs/ElunaDoc/templates/method.html

I haven't changed it since we forked Eluna, and even there I think it was based on an old version of the Rust documentation. So yeah the code shows its age

Reply:
ok